extensions/standard-processors/tests/unit/ManifestTests.cpp

```diff
@@ -74,15 +74,13 @@ TEST_CASE("Test Valid Regex", "[validRegex]") {
   const auto &prop_descriptors = proc_0.children[0];
   REQUIRE_FALSE(prop_descriptors.children.empty());
   const auto &prop_0 = prop_descriptors.children[0];
-  REQUIRE(prop_0.children.size() >= 3);
+  REQUIRE(prop_0.children.size() >= 6);
```

Comment:
what changed here?

Reply:
We were dereferencing `children[5]` and `children[6]`, but only checked that `children` has at least 3 elements.